dev-env/README.md

```diff
@@ -10,19 +10,6 @@ of your git clone, which runs MetalLB in an unconfigured state:
 inv dev-env
 ```
 
```

Comment:
We will still need to be able to run the development environment in order to manually test and try things out.